test/unit/controllers/ApplicationControllerSpec.scala

```diff
@@ -231,6 +250,31 @@ class ApplicationControllerSpec extends ControllerSpec with BeforeAndAfterEach w
         await(result).header.headers("Location") should endWith("unable-to-use-id")
         verify(mockSessionCache).saveEori(any())(any())
       }
+
+      "enrolment is not in use but user is Assistant" in {
```

Comment:
So what is the expected behaviour? Test title should really be a bit more descriptive, please.

Reply:
done

Reply:
These test cases are structured in a bit different way, the expectations are set at the top if you extend the changes and scroll above.